portmgr-template.adoc

```diff
@@ -2,7 +2,7 @@
 
 Links: +
 link:https://www.FreeBSD.org/ports/[About FreeBSD Ports] URL:link:https://www.FreeBSD.org/ports/[https://www.FreeBSD.org/ports/] +
-link:https://docs.freebsd.org/en/articles/contributing/ports-contributing/[Contributing to Ports] URL: link:https://docs.freebsd.org/en/articles/contributing/ports-contributing/[https://docs.freebsd.org/en/articles/contributing/ports-contributing/] +
+link:https://docs.freebsd.org/en/articles/contributing/#ports-contributing[Contributing to Ports] URL: link:https://docs.freebsd.org/en/articles/contributing/#ports-contributing[https://docs.freebsd.org/en/articles/contributing/#ports-contributing] +
 link:http://portsmon.freebsd.org/[FreeBSD Ports Monitoring] URL: link:http://portsmon.freebsd.org/[http://portsmon.freebsd.org/] +
 link:https://www.freebsd.org/portmgr/[Ports Management Team] URL: link:https://www.freebsd.org/portmgr/[https://www.freebsd.org/portmgr/] +
 link:http://ftp.freebsd.org/pub/FreeBSD/ports/ports/[Ports Tarball] URL: link:http://ftp.freebsd.org/pub/FreeBSD/ports/ports/[http://ftp.freebsd.org/pub/FreeBSD/ports/ports/]
```

Comment:
Not sure it's worth changing, but in case you're tempted to, https: works here, at least for the index page. I haven't tried downloading any of the tarballs.